net-dns/knot/knot-2.3.1.ebuild

```diff
@@ -35,10 +35,6 @@ DEPEND="${RDEPEND}
 	doc? ( dev-python/sphinx )
 "
 
-PATCHES=(
-	"${FILESDIR}/2.3.0-added-conversion-to-wire-format-when-writing-FNV64-hash-values.patch"
-)
-
 S="${WORKDIR}/${P/_/-}"
 
 src_configure() {
```

Comment:
in `src_configure`, please change
`$(usex systemd --enable-systemd=yes --enable-systemd=no)`
to
`--enable-systemd=$(usex systemd)`